### PR TITLE
make pep8 compulsory

### PR DESCRIPTION
**Pull request by Gustry**

### CODE REVIEW

**Comment by Gustry** [comment]
I'm just discovering that this plugin got flak8. Feel free to cose this PR if it's not relevant. I think we could improve flake8 maybe

File: .flake8 [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [flake8]
 exclude = */docs/*,*/.tox/*,*/.venv/*
-max-line-length = 100
+max-line-length = 89
```
Comment (ismailsunni) [inline .flake8]: Why is it set to 89? no 80?

File: .flake8 [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
 # E12x continuation line indentation
 # E251 no spaces around keyword / parameter equals
 # E303 too many blank lines (3)
-ignore = E126,E251,E303
+ignore = E126,E251,E303,E125
```
Comment (ismailsunni) [inline .flake8]: May be you can add the explanation of E125 like the others in the lines above

File: .flake8 (hunk 1 above)
Comment (Gustry) [inline .flake8]: Ah yes, typo. I guess it should be 79, no ?

File: .flake8 (hunk 2 above)
Comment (Gustry) [inline .flake8]: The same as line 5: `# E12x continuation line indentation`

Comment (ismailsunni) [inline .flake8]: ah yes, I didn't see the `x` :D

File: .flake8 (hunk 1 above)
Comment (timlinux) [inline .flake8]: Can you tweak this and then merge @Gustry ?

**Comment by Gustry** [comment]
I'm moving from pep8 to flake8. I will merge this one soon @timlinux

**Comment by cchristelis** [comment]
@Gustry, thanks for this.

**Comment by Gustry** [comment]
We should add this flake8 test on every python repository. I will merge this PR today, Travis should be happy after that.

**Pull request merged**